openeo/local/processing.py

```diff
@@ -59,4 +68,74 @@ def load_local_collection(*args, **kwargs):
 PROCESS_REGISTRY["load_collection"] = Process(
     spec=openeo_processes_dask.specs.load_collection, 
     implementation=load_local_collection,
+)
+
+def resample_cube_spatial_rioxarray(*args, **kwargs):
+    pretty_args = {k: type(v) for k, v in kwargs.items()}
+    _log.debug(f"Running process resample_cube_spatial")
+    _log.debug(f"kwargs: {pretty_args}")
+    _log.debug("-" * 80)
```

Comment:
don't log like you would `print` (e.g. multiple lines, `"-" * 80`) , keep things that belong together in same log call

Reply:
I was copying the style of the implemented processes in `openeo-processes-dask`. @LukeWeidenwalker any thought on this?

Reply:
Agree with @soxofaan, this is something that could be improved in openeo-processes-dask too!

Reply:
I will improve this once we agree in `openeo-processes-dask` how to improve the logs format

Reply:
Proposal here: https://github.com/Open-EO/openeo-processes-dask/pull/128

Reply:
Modified according to the new format implemented by @LukeWeidenwalker